lms/djangoapps/courseware/grades.py

```diff
@@ -392,7 +392,7 @@ def _grade(student, course, keep_raw_scores):
 
     More information on the format is in the docstring for CourseGrader.
     """
-    course_structure = get_course_blocks(student, course.location)
+    course_structure = getattr(course, 'structure', [])
```

Comment:
How do we feel about this vs the defensive approach taken in the original commit (https://github.com/edx/edx-platform/pull/12858/commits/b15d4e23fa1558cac1acfcdbd878dd876a4b68fb)?

Should we add the `if not len(course_structure): return None` logic that `_progress_summary` uses here?